reference/format/dashboard.rst

```diff
@@ -0,0 +1,189 @@
+
+``dashboards``
+--------------
+
+**type**: ``map``
+
+The ``dashboards`` key is used to configure options for dashboards. The key is the name of the
+dashboard and the options are the values. Currently, there is only one supported option:
+
+.. _reference-format-dashboard-twig:
+
+``twig``
+~~~~~~~~
+
+**type**: ``string``
+
+The Twig template used to render the dashboard. ``OroDashboardBundle:Index:default.html.twig`` is
+used when no value is configured.
+
+``widgets``
+-----------
+
+**type**: ``map``
+
+The keys are strings that each uniquely identiy a dashboard widget. The value again is a map that
+holds the configuration of the widget for which the following options can be used:
+
+``acl``
+~~~~~~~
+
+**type**: ``string``
+
+The given access control list is evaluated to decide whether or not a user is allowed to see the
+widget.
+
+``configuration``
+~~~~~~~~~~~~~~~~~
+
+**type**: ``map``
+
+You can define an arbitrary set of configuration options the user can use to configure the widget
+(for example to customize the way it is rendered, its behavior, etc.). Each key of the map
+represents the name of a configuration option, its values are maps that can use the following keys:
+
+``options`` (**type**: ``map``)
+
+    A map of options and their values configuring the Symfony form type (see the ``type`` option
+    below). You can find a complete reference of available options for Symfony's built-in form
+    types `in the Symfony documentation`_.
+
+``show_on_widget`` (**type**: ``boolean``)
+
+    If set to ``true``, the user can edit the option at the bottom of the widget. Otherwise, the
+    value can be modified by clicking on the "wheel" icon when the widget is rendered on the
+    dashboard.
+
+``type`` (**type**: ``string``)
+
+    The Symfony form type name that is used to define what kind of input will be used to ask for
+    user input.
+
+``description``
+~~~~~~~~~~~~~~~
+
+**type**: ``string``
+
+A description that is used to explain what the widget is supposed to do.
+
+``icon``
+~~~~~~~~
+
+**type**: ``string``
+
+This option configures an icon that is shown next to the widget's name and description in the UI
+when the users configures a dashboard.
+
+``label``
+~~~~~~~~~
+
+**type**: ``string``
+
+The widget name.
+
+``position``
+~~~~~~~~~~~~
+
+**type**: ``integer`` **default**: ``0``
+
+The initial position of a widget when it is added to a dashboard.
+
+``route``
+~~~~~~~~~
+
+**type**: ``string``
+
+When the widget is rendered on the dashboard, the URL of the configured route will be requested and
+the response will be displayed at the widget's position.
+
+``route_parameters``
+~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``map``
+
+Additional parameters that are passed to the URL generator when generating the URL for the
+configured route.
+
+``widgets_configuration``
+-------------------------
+
+**type**: ``map``
+
+With the `configuration`_ key you can define a set of options the user can customize for a certain
```

Comment:
Here you describe `widgets_configuration` parameter, not the `configuration` for current widget. Please correct the description